node-util/sbin/oo-init-quota

```diff
@@ -49,12 +49,12 @@ function get_mountpoint() {
 
 # find the filesystem type above a given file or directory
 function get_filesystem_type() {
-    egrep "\s+$1\s+" /etc/fstab | awk '{ print $3 }'
+    egrep "^[^#].+\s+$1\s+" /etc/fstab | awk '{ print $3 }'
 }
 
 # get the fstab options for a given mount point
 function get_fstab_mount_options() {
-    cat /etc/fstab | tr -s ' ' | grep " $1 "| awk '{print $4;}'
+    egrep "^[^#].+\s+$1\s+" /etc/fstab | awk '{ print $4 }'
 }
```

Comment:
Sure, I can do that.

The `[^#]` means anything but a `#` character. The regex will match any line that begins with anything other than `#`.